nest/03.nest.md

```diff
@@ -0,0 +1,89 @@
+# 2 nest
+In this chapter, we will detail how to use the binary `nest`.
+As explained before, this binary is mainly designed for ease of use.
+However, anyone can use it.
+
+## 2.1 nest options
+`nest` has three options available:
+
+* `-h` or `--help`, which prints the help information of `nest`, consisting of a short explanation for each of `nest`'s options and commands.
+* `-v`, which sets the level of verbosity for `nest`. Basically, how much `nest` will explain what it's doing in the terminal.
```

Comment:
This sentence implies that there are several levels of verbosity that can be set, but doesn't really explain how to choose one over another

Reply:
(Would be cool to actually use the flag, now that it's there :o)